### PR TITLE
chunk:add more benchmarks

### PR DESCRIPTION
**Pull request by zhexuany**

### CODE REVIEW

**Comment by zhexuany** [comment]
@coocood @zz-jason PTAL

File: util/chunk/chunk_test.go [hunk 1]
```diff
@@ -501,6 +501,101 @@ func appendRow(chk *Chunk, row Row) {
 	}
 }
 
+func BenchmarkAppendBytes1024(b *testing.B) {
+	chk := NewChunk([]*types.FieldType{types.NewFieldType(mysql.TypeString)})
+	var bs = make([]byte, 10, 1000)
```
Comment (zz-jason) [inline util/chunk/chunk_test.go]: `bs := make([]byte, 10)`

File: util/chunk/chunk_test.go [hunk 2]
```diff
@@ -501,6 +501,101 @@ func appendRow(chk *Chunk, row Row) {
...
+	for i := 0; i < b.N; i++ {
+		appendBytes(chk, bs, 1024)
+	}
+}
+
+func BenchmarkAppendBytes512(b *testing.B) {
+	chk := NewChunk([]*types.FieldType{types.NewFieldType(mysql.TypeString)})
+	var bs = make([]byte, 10, 1000)
```
Comment (zz-jason) [inline util/chunk/chunk_test.go]: ditto

**Comment by coocood** [comment]
LGTM

**Pull request merged**